integration-test/playwright.config.ts

```diff
@@ -0,0 +1,11 @@
+import {defineConfig, devices} from '@playwright/test';
+
+export default defineConfig({
+  globalSetup: require.resolve('./global-setup'),
```

Comment:
We might not need `globalSetup` here. Since the baseURL can be set up in here directly by providing `use` property:
```
  use: {
    baseURL: process.env.API_HOST || 'http://localhost:8080',
  },
```
Also, project should be named a little differently, something like `api_tests` and remove `use`, since we don't use browser here.

Reply:
Ahh this baseURL, will do the tricky, so I don't need to pass in all tests, nice!!